### PR TITLE
SSE version of PresetOutputs::PerPixelMath

### PR DESCRIPTION
**Pull request by mbellew**
tested using 320x240, yin - 315 - Ocean of Light (yo im peakin yo Eo.S.-Phat).milk just about doubled FPS from 60ish to 110ish.

From google-pprof, doesn't entirely match up with my FPS observation, but does show CPU usage going down.

before
     881  48.6%  48.6%      881  48.6% __iscanonicall
     453  25.0%  73.7%     1225  67.6% PresetOutputs::PerPixelMath
after
     837  47.0%  47.0%      837  47.0% __iscanonicall
     131   7.4%  54.4%      965  54.2% PresetOutputs::PerPixelMath_sse

### CODE REVIEW

**Comment by revmischa** [comment]
Build is failing on OSX https://travis-ci.org/projectM-visualizer/projectm/jobs/381463843

**Comment by mbellew** [comment]
I have an old iMac mini around somewhere I can blow the dust off and fire up...

**Comment by revmischa** [comment]
If it's easier you can make changes and push them and then look at travis. It takes 5-10 minutes to run through usually.

File: src/libprojectM/wipemalloc.cpp [hunk 1]
```diff
@@ -25,21 +25,61 @@
...
+    if (allocated == NULL)
+    {
+        printf( "wipe_aligned_malloc() failed to allocate %d bytes\n", (int)size );
+        return NULL;
+    }
+    void *ret = (void*) (((size_t)allocated + sizeof(void*) + align -1) & ~(align-1));
+    *((void**)((size_t)ret - sizeof(void*))) = allocated;
+    return ret;
+#else
+    void *mem = aligned_alloc( align, size );
```
Comment (revmischa) [inline src/libprojectM/wipemalloc.cpp]: is aligned_alloc available on EVERY other platform that isn't apple? raspi? BSD? windows?
I know mac is special and needs ^2 alignment, that's fine. i'm concerned about the aligned_alloc being portable.

Comment (mbellew) [reply src/libprojectM/wipemalloc.cpp]: I honestly don't know.  It is C11 standard, and the only google references I found about lack of support seem related to OSX.  I could remove the #ifdef and always use the hand-written version.

Comment (revmischa) [reply src/libprojectM/wipemalloc.cpp]: Well do whatever you think is best

Comment (exp) [reply src/libprojectM/wipemalloc.cpp]: MacOS seems to contain posix_memalign, would that not be sufficient? It seems to work (almost) exactly the same except takes an address to write the pointer to.

Comment (mbellew) [reply src/libprojectM/wipemalloc.cpp]: There's this from stackoverflow

**Comment by revmischa** [comment]
I have one error compiling w/ xcode:

screen shot 2018-05-22 at 22 03 23

**Pull request merged**